### PR TITLE
adjust most svg size to rem

### PR DESCRIPTION
**Pull request by fiji-flo**

### CODE REVIEW

**Comment by andrew-sunada** [comment]
Yea I definitely like that align-items: center change as well.

File: src/components/ui/Button.vue [hunk 1]
```diff
@@ -33,10 +33,14 @@ export default {
   color: var(--blue-60);
   text-decoration: none;
 }
+.button > * {
+  align-self: center;
```
[the remark this answers is not in the document]
Comment (fiji-flo) [reply src/components/ui/Button.vue]: Indeed.

**Comment by andrew-sunada** [comment]
When the text on the blue buttons breaks line, the arrow is still at the top. Also the main button on the main page "My profile" looks off with the arrow.
Screen Shot 2019-10-31 at 9 15 13 AM
Screen Shot 2019-10-31 at 9 15 34 AM

**Comment by fiji-flo** [comment]
> When the text on the blue buttons breaks line, the arrow is still at the top. Also the main button on the main page "My profile" looks off with the arrow.
> Screen Shot 2019-10-31 at 9 15 13 AM
> Screen Shot 2019-10-31 at 9 15 34 AM

For me the ContactMe button looks fine with the latest changes:
image

Good catch on the home page button. That one overrides `display`.

**Pull request merged**